WebRoot/js/zimbraMail/mail/view/prefs/ZmMailPrefsPage.js

```diff
@@ -834,40 +859,36 @@ function() {
 	this._numUsedText.innerHTML = AjxMessageFormat.format(ZmMsg.whiteBlackNumUsed, [this._listView.size(), this._max]);
 };
 
+ZmWhiteBlackList.prototype._SelectionListener = function (ev) {
+	var selection = this._listView.getSelection();
+	if (ev && ev.target && this._listView.delButtons[ev.target.id]) {
```

Comment:
As discussed, for this, I referred to handling of removing of contacts in contacts group view.
Please refer function, 
`this._groupMembersListView.addSelectionListener(new AjxListener(this, this._groupMembersSelectionListener));`
defined in ZmGroupview.js.